### PR TITLE
DOCS-1398,DOCS-1399: Update README

### PR DESCRIPTION
**Pull request by andf-viam**
Update README to be inline with established MR readme structure, as seen in the [Agilex LIMO readme](https://github.com/viam-labs/agilex/blob/main/README.md).
- Merged all content from [Docs reference page](https://docs.viam.com/registry/examples/odrive/) to this readme.
- Will delete Docs reference page in separate PR ... to separate repo.

Happy to make any changes!

### CODE REVIEW

**Comment by andf-viam** [comment]
Hi @martha-johnston , Docs is sweeping through module readmes and updating with a standardizes set of headings and other info. We will then remove [our version of this on docs.viam.com](https://docs.viam.com/registry/examples/odrive/) with this same info.

Let me know if you'd like to see any changes, if I've missed anything in my conversion here, or if you'd rather I ping a different engineer. Thanks so much!

File: README.md [hunk 1]
```diff
@@ -1,84 +1,73 @@
...
+### Configure the ODrive motor
 
-    * Update the `executable_path` (string) to the location of `run.sh` on your machine
-    * (For the `canbus`  model) update the `canbus_node_id` (int) to the node ID of whichever CAN node you'd like to use
-    
-Copy and paste this JSON configuration for your robot on [the Viam app](https://app.viam.com): 
+> [!NOTE]  
+> Before configuring your motor, you must [create a robot](https://docs.viam.com/manage/fleet/robots/#add-a-new-robot).
+
+Navigate to the **Config** tab of your robot’s page in [the Viam app](https://app.viam.com/). Click on the **Components** subtab and click **Create component**. Select the `base` type, then select the `agilex-limo` model. Enter a name for your base and click **Create**.
```
Comment (martha-johnston) [inline README.md:41]: assuming this was a copy/paste line that wasn't edited, should be motor and odrive not base and agilex

Comment (andf-viam) [reply README.md]: Thanks for catching!

File: README.md [hunk 2]
```diff
@@ -1,84 +1,73 @@
...
+   * Note that `iq_msg_rate_ms` in the config defaults to `0`, and you **must** set this to around `100` to use the [motor API's `SetPower` method](https://docs.viam.com/components/motor/#setpower).
+   * See the section [Add an `odrive_config_file`](#add-an-odrive_config_file) for more information on dynamic configuration.
+* See the [ODrive CAN documentation](https://docs.odriverobotics.com/v/latest/can-guide.html) for detailed information on how to set up CAN on your ODrive.
+
+### Connect your ODrive to your single-board computer
+
+Connect your ODrive motor to your single-board computer in one of the following ways:
+
+* For a `serial` connection: plug the [USB Isolator for ODrive](https://odriverobotics.com/shop/usb-c-to-usb-a-cable-and-usb-isolator) into a USB port on your board, and then plug a USB-C to USB-A cable from the isolator to the ODrive.
```
Comment (martha-johnston) [inline README.md:31]: maybe switch USB-C and USB-A in this sentence so it's in the same order as isolator and Odrive. Meaning USB-A goes to the isolator and USB-C goes to the ODrive

File: README.md [hunk 3]
```diff
@@ -1,84 +1,73 @@
...
+### Configure the ODrive motor
 
-    * Update the `executable_path` (string) to the location of `run.sh` on your machine
-    * (For the `canbus`  model) update the `canbus_node_id` (int) to the node ID of whichever CAN node you'd like to use
-    
-Copy and paste this JSON configuration for your robot on [the Viam app](https://app.viam.com): 
+> [!NOTE]  
+> Before configuring your motor, you must [create a robot](https://docs.viam.com/manage/fleet/robots/#add-a-new-robot).
+
+Navigate to the **Config** tab of your robot’s page in [the Viam app](https://app.viam.com/). Click on the **Components** subtab and click **Create component**. Select the `motor` type, then select the `odrive:serial` or `odrive:canbus` model. Enter a name for your base and click **Create**.
```
Comment (martha-johnston) [inline README.md:41]: oops. also "Enter a name for your motor"

File: README.md [hunk 4]
```diff
@@ -138,6 +128,12 @@ An alternative to adding an `odrive_config_file` is running the command `odrivet
 }
 ```
 
+## Next Steps
+
+- To test your ODrive motor, go to the [**Control** tab](https://docs.viam.com/manage/fleet/robots/#control).
+- To write code against your ODrive motor, use one of the [available SDKs](https://docs.viam.com/program/).
+- To view examples using a motor component, explore [these tutorials](https://docs.viam.com/tutorials/).
+  
 ## Troubleshooting
```
Comment (martha-johnston) [inline README.md:138]: also a question here, is the troubleshooting section now empty? maybe it's just on the website and not in the readme? I remember that having important stuff. if it's only on the online documentation I think we should transfer it here

**Comment by andf-viam** [comment]
@martha-johnston thanks for the great feedback! Appreciated!

**Review by martha-johnston — APPROVED** [review]
lgtm

**Comment by npentrel** [comment]
@andf-viam this looks good but I think we're missing the serial example. I think in this case we can skip the template because it's so similar to the example but adding the serial one might be good?

**Comment by andf-viam** [comment]
Thanks for helpful reviews, all!

**Pull request merged**